jenkins-scm-koji-plugin/src/main/java/hudson/plugins/scm/koji/client/KojiBuildDownloader.java

```diff
@@ -267,6 +259,15 @@ public void println(String s) {
         return l;
     }
 
+    private GlobPredicate getGlobPredicate(String subpackageBlacklist) {
```

Comment:
Instead of this, the KojiBuildDownloader.java itself can implement the interface and have the method implicit. As it is the `currentListener.getLogger().println(s);` itself

Reply:
Pls continue in #280